build.make

```diff
@@ -51,7 +51,7 @@ IMAGE_TAGS+=$(shell git branch -r --points-at=HEAD | grep 'origin/release-' | gr
 IMAGE_TAGS+=$(shell tagged="$$(git describe --tags --match='v*' --abbrev=0)"; if [ "$$tagged" ] && [ "$$(git rev-list -n1 HEAD)" = "$$(git rev-list -n1 $$tagged)" ]; then echo $$tagged; fi)
 
 # Images are named after the command contained in them.
-IMAGE_NAME=$(REGISTRY_NAME)/$*
+IMAGE_NAME?=$(REGISTRY_NAME)/$*
```

Comment:
But why this?

Reply:
mainly I want to:
```
export REGISTRY_NAME=...
export IMAGE_NAME=...
make build
make container
```

with this PR, it could tag image name with correct registry name, otherwise need workaround, thanks.

Reply:
But why set `IMAGE_NAME`? It's a combination of other input variables. You shouldn't need to set it.

Reply:
I think there is var name conflict in NFS Makefile, reverted this change. PTAL.